src/models/contact/person/index.js

```diff
@@ -50,3 +69,85 @@ export function createPersonDatasetUrl(addressBook, id = uuid()) {
 export async function getPersonAll(addressBook, fetch) {
   return getContactAll(addressBook, [PERSON_CONTACT], fetch);
 }
+
+export function getWebIdUrl(dataset, iri) {
+  const thing = getThing(dataset, iri);
+  const webIdNodeUrl = getUrl(thing, vcard.url);
+  if (webIdNodeUrl) {
+    const webIdNode = getThing(dataset, webIdNodeUrl);
+    return webIdNode && getUrl(webIdNode, vcard.value);
+  }
+  return getUrl(thing, foaf.openid);
+}
+
+export function createWebIdNode(webId, iri) {
+  const webIdNode = chain(
+    createThing(),
+    (t) => addUrl(t, rdf.type, vcardExtras("WebId")),
+    (t) => addUrl(t, vcard.value, webId)
+  );
+  const webIdNodeUrl = asUrl(webIdNode, iri);
+  return { webIdNode, webIdNodeUrl };
+}
+
+export async function createPersonContact(addressBookContainerUrl, contact) {
```

Comment:
Seeing this now I realize we need to coordinate a bit with `saveContact` in `src/models/contact/index.js`. Let's have a chat on this.